vscode-dotnet-runtime-library/distro-data/distro-support.json

```diff
@@ -1,17 +1,90 @@
 {
 	"$schema": "../../Documentation/json-schema/distro-support-schema.json",
 	"Ubuntu": {
-		"installCommand": "sudo apt-get update && sudo apt-get install -y {0}",
-		"uninstallCommand": "sudo apt-get remove {0}",
-		"updateCommand": "sudo apt-get update && apt-get upgrade -y {0}",
-		"isInstalledCommand": "sudo apt list --installed {0}",
-		"packageLookupCommand": "dpkg -l {0}",
+		"installCommand":
+		[
+			{
+				"runUnderSudo": true,
+				"commandRoot": "apt-get",
+				"commandParts": ["update"]
+			},
+			{
+				"runUnderSudo": true,
+				"commandRoot": "apt-get",
+				"commandParts": ["install", "-y", "{0}"]
```

Comment:
These `{0}` placeholders are .NET format string style, but they don't give any semantic meaning for review - should we have a named placeholder like `{packageName}` instead that we can explicitly substitute in the command processor?

Reply:
Hm, I like this placeholder idea a lot better. Done! Thanks for the suggestion.